modules/api/src/main/java/org/eclipse/xpanse/api/controllers/ServiceConfigurationApi.java

```diff
@@ -51,6 +52,20 @@ public class ServiceConfigurationApi {
     @Resource
     private ServiceConfigurationManager serviceConfigurationManager;
 
+    @Tag(name = "ServiceConfiguration",
+            description = "APIs for managing service's configuration.")
+    @GetMapping(value = "/service/current/config/{serviceId}",
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(description = "Query the service's configuration by serviceId.")
```

Comment:
Itis better to update the description with 'Query the service's  current configuration by id of the deployed service'